### DISCUSSION TITLE
How can I set a run timeout for a graph asset?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18089] [Q&A]
Hello,

Maybe somebody knows how to properly set a run timeout for graph assets?

There is an <https://docs.dagster.io/deployment/run-monitoring#run-cancelation-timeouts|example> of how to that in the job level, but unfortunately <https://docs.dagster.io/_apidocs/assets#dagster.graph_asset|graph asset> does not except tags parameter.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700126136286599?thread_ts=1700126136.286599&cid=C01U954MEER).*

**Comment by alangenfeld** [comment]
Run timeouts can currently only be set at the job level.
